src/ledger.ts

```diff
@@ -161,4 +172,11 @@ export class LedgerCanister {
     // Successful tx. Return the block height.
     return BigInt(PbBlockHeight.deserializeBinary(responseBytes).getHeight());
   };
+
+  private subAccountIdToSubaccount = (index: number): Subaccount => {
```

Comment:
`index: number` is really an index or it's an `id`?

calls = `this.subAccountIdToSubaccount(fromSubAccountId)`

declaration = `subAccountIdToSubaccount = (index: number)`

it's a bit misleading I would say. can you unify the naming?

Reply:
good point, I copied the helper `ts` but it makes more sense `subAccountId` than `index`

Reply:
done